<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Конвертер для МК</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Изображение</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>611</width>
          <height>401</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="ScaledPixmap" name="ImageToDisplay">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>58</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>300</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="textToDisplay">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>60</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="lineWidth">
              <number>5</number>
             </property>
             <property name="midLineWidth">
              <number>4</number>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetFixedSize</enum>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButtonOpen">
               <property name="toolTip">
                <string>Открыть файл (изображение)</string>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">#pushButtonOpen {
	border: 2px solid black;
	border-radius: 10px;
}

#pushButtonOpen:hover{ 
	border: 2px solid black;
	border-radius: 10px;
	background-color: rgb(232,95,76);
} 
#pushButtonOpen:pressed{
	border: 2px solid black;
	border-radius: 10px; 
	background-color: rgb(0, 129, 4);
}

</string>
               </property>
               <property name="text">
                <string>Открыть</string>
               </property>
               <property name="icon">
                <iconset resource="Resource.qrc">
                 <normaloff>:/Icon/Icon/kisspng-computer-icons-clip-art-icon-instagram-5b396cec732b10.9586206015304900924717.png</normaloff>:/Icon/Icon/kisspng-computer-icons-clip-art-icon-instagram-5b396cec732b10.9586206015304900924717.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonConvert">
               <property name="styleSheet">
                <string notr="true">#pushButtonConvert {
	border: 2px solid black;
	border-radius: 10px;
}

#pushButtonConvert:hover{ 
	border: 2px solid black;
	border-radius: 10px;
	background-color: rgb(232,95,76);
} 
#pushButtonConvert:pressed{
	border: 2px solid black;
	border-radius: 10px; 
	background-color: rgb(0, 129, 4);
}</string>
               </property>
               <property name="text">
                <string>Конвертировать</string>
               </property>
               <property name="icon">
                <iconset resource="Resource.qrc">
                 <normaloff>:/Icon/Icon/kisspng-arrow-emoji-computer-icons-right-arrow-5ad6ec2b7ac819.6936564815240346035029.png</normaloff>:/Icon/Icon/kisspng-arrow-emoji-computer-icons-right-arrow-5ad6ec2b7ac819.6936564815240346035029.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Шрифт</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Дополнительные символы</string>
            </property>
            <property name="textFormat">
             <enum>Qt::MarkdownText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">@charset &quot;utf-8&quot;;
</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>4</number>
            </property>
            <property name="midLineWidth">
             <number>4</number>
            </property>
            <property name="tabChangesFocus">
             <bool>false</bool>
            </property>
            <property name="documentTitle">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="plainText">
             <string/>
            </property>
            <property name="backgroundVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>Unicode кирилические символы (0410 - 044F)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_2">
              <property name="text">
               <string>Unicode латинские символы (0020 - 007E)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_3">
              <property name="text">
               <string>Unicode Дополнительные символы</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="dispImageFont">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButtonOpenFont">
            <property name="toolTip">
             <string>Открыть файл (изображение)</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButtonOpenFont {
	border: 2px solid black;
	border-radius: 10px;
}

#pushButtonOpenFont:hover{ 
	border: 2px solid black;
	border-radius: 10px;
	background-color: rgb(232,95,76);
} 
#pushButtonOpenFont:pressed{
	border: 2px solid black;
	border-radius: 10px; 
	background-color: rgb(0, 129, 4);
}

</string>
            </property>
            <property name="text">
             <string>Шрифт</string>
            </property>
            <property name="icon">
             <iconset resource="Resource.qrc">
              <normaloff>:/Icon/Icon/book_closed_icon_143831.png</normaloff>:/Icon/Icon/book_closed_icon_143831.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonOpenColor">
            <property name="toolTip">
             <string>Открыть файл (изображение)</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButtonOpenColor {
	border: 2px solid black;
	border-radius: 10px;
}

#pushButtonOpenColor:hover{ 
	border: 2px solid black;
	border-radius: 10px;
	background-color: rgb(232,95,76);
} 
#pushButtonOpenColor:pressed{
	border: 2px solid black;
	border-radius: 10px; 
	background-color: rgb(0, 129, 4);
}

</string>
            </property>
            <property name="text">
             <string>Цвет фона</string>
            </property>
            <property name="icon">
             <iconset resource="Resource.qrc">
              <normaloff>:/Icon/Icon/book_closed_icon_143831.png</normaloff>:/Icon/Icon/book_closed_icon_143831.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonOpenColorFont">
            <property name="toolTip">
             <string>Открыть файл (изображение)</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButtonOpenColorFont {
	border: 2px solid black;
	border-radius: 10px;
}

#pushButtonOpenColorFont:hover{ 
	border: 2px solid black;
	border-radius: 10px;
	background-color: rgb(232,95,76);
} 
#pushButtonOpenColorFont:pressed{
	border: 2px solid black;
	border-radius: 10px; 
	background-color: rgb(0, 129, 4);
}

</string>
            </property>
            <property name="text">
             <string>Цвет шрифта</string>
            </property>
            <property name="icon">
             <iconset resource="Resource.qrc">
              <normaloff>:/Icon/Icon/book_closed_icon_143831.png</normaloff>:/Icon/Icon/book_closed_icon_143831.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonOpenText">
            <property name="toolTip">
             <string>Открыть файл (изображение)</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButtonOpenText {
	border: 2px solid black;
	border-radius: 10px;
}

#pushButtonOpenText:hover{ 
	border: 2px solid black;
	border-radius: 10px;
	background-color: rgb(232,95,76);
} 
#pushButtonOpenText:pressed{
	border: 2px solid black;
	border-radius: 10px; 
	background-color: rgb(0, 129, 4);
}

</string>
            </property>
            <property name="text">
             <string>Открыть</string>
            </property>
            <property name="icon">
             <iconset resource="Resource.qrc">
              <normaloff>:/Icon/Icon/book_closed_icon_143831.png</normaloff>:/Icon/Icon/book_closed_icon_143831.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonConvertText">
            <property name="styleSheet">
             <string notr="true">#pushButtonConvertText {
	border: 2px solid black;
	border-radius: 10px;
}

#pushButtonConvertText:hover{ 
	border: 2px solid black;
	border-radius: 10px;
	background-color: rgb(232,95,76);
} 
#pushButtonConvertText:pressed{
	border: 2px solid black;
	border-radius: 10px; 
	background-color: rgb(0, 129, 4);
}</string>
            </property>
            <property name="text">
             <string>Конвертировать</string>
            </property>
            <property name="icon">
             <iconset resource="Resource.qrc">
              <normaloff>:/Icon/Icon/kisspng-arrow-emoji-computer-icons-right-arrow-5ad6ec2b7ac819.6936564815240346035029.png</normaloff>:/Icon/Icon/kisspng-arrow-emoji-computer-icons-right-arrow-5ad6ec2b7ac819.6936564815240346035029.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openMenu"/>
    <addaction name="exitMenu"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="convertnMenu"/>
    <addaction name="configMenu"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Помощь</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openMenu">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Icon/Icon/kisspng-computer-icons-clip-art-icon-instagram-5b396cec732b10.9586206015304900924717.png</normaloff>:/Icon/Icon/kisspng-computer-icons-clip-art-icon-instagram-5b396cec732b10.9586206015304900924717.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="exitMenu">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Icon/Icon/kisspng-computer-icons-running-download-clip-art-sport-activities-running-icon-5ab063fbaf7c06.3929452415215093717188.png</normaloff>:/Icon/Icon/kisspng-computer-icons-running-download-clip-art-sport-activities-running-icon-5ab063fbaf7c06.3929452415215093717188.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="convertnMenu">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Icon/Icon/kisspng-arrow-emoji-computer-icons-right-arrow-5ad6ec2b7ac819.6936564815240346035029.png</normaloff>:/Icon/Icon/kisspng-arrow-emoji-computer-icons-right-arrow-5ad6ec2b7ac819.6936564815240346035029.png</iconset>
   </property>
   <property name="text">
    <string>Конвертровать</string>
   </property>
  </action>
  <action name="configMenu">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Icon/Icon/kisspng-gear-sprocket-clip-art-gears-vector-5add084d6a9e11.5447134915244350214367.png</normaloff>:/Icon/Icon/kisspng-gear-sprocket-clip-art-gears-vector-5add084d6a9e11.5447134915244350214367.png</iconset>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScaledPixmap</class>
   <extends>QLabel</extends>
   <header>scaledpixmap.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
